app/controllers/catalog_controller.rb

```diff
@@ -707,7 +707,7 @@ def ot_copy_submit
 
   def ot_remove_submit
     assert_privileges("orchestration_template_remove")
-    checked = find_checked_items
+    checked = find_checked_ids_with_rbac(OrchestrationTemplate)
     checked[0] = params[:id] if checked.blank? && params[:id]
     elements = OrchestrationTemplate.where(:id => checked)
```

Comment:
Hm, looking at this, I'm thinking using `find_checked_records_with_rbac` may be better here. ☝️